.gitignore

```diff
@@ -10,3 +10,8 @@ dmoj/local_settings.py
 resources/style.css
 resources/martor-description.css
 sass_processed
+dmojsite
+dmojsite/*
+problems
+problems/*
+*.log
```

Comment:
These lines seem specific to your install. It's recommended to not store unrelated files inside the repository anyways. Would remove this from the PR.

Reply:
Sure! Removing.